schema/gluu_schema.json

```diff
@@ -6069,7 +6146,46 @@
                 "top"
             ], 
             "x_origin": "Gluu created objectclass"
-        } 
+        },
+        {
+            "kind": "STRUCTURAL",
+            "may": [
+                "key",
+                "value",
+                "createdAt",
+                "expiredAt",
+                "type"
```

Comment:
@duttarnab Instead of creating new attributes I propose to reuse existing one which are already used for this purpose.
E.g. `key`->`oxId`, `value` -> `dat` (`dat` because we tried to save each letter for couchbase), `createdAt`->`iat` and so on.
What do you think? I think we should re-use attributes instead of duplication. I hope in 5.x version we completely revise our attributes and remove all duplicates (as well as optimize names)